src/vs/workbench/parts/emmet/node/actions/expandAbbreviation.ts

```diff
@@ -31,7 +31,8 @@ class ExpandAbbreviationAction extends BasicEmmetEditorAction {
 					EditorContextKeys.hasOnlyEmptySelection,
 					EditorContextKeys.hasSingleSelection,
 					EditorContextKeys.tabDoesNotMoveFocus,
-					ContextKeyExpr.has('config.emmet.triggerExpansionOnTab')
+					ContextKeyExpr.has('config.emmet.triggerExpansionOnTab'),
+					ContextKeyExpr.not('config.emmet.autocomplete')
```

Comment:
The emmet auto complete feature will be coming from the extension.
So at present I have the this (`emmet.autocomplete`) setting being contributed by the extension with default value as `true`.

Therefore, I did not register this setting like the way it is in the link you mentioned.